apps/sla/time.py

```diff
@@ -24,6 +24,35 @@ def add_days(dt: datetime, days: int) -> datetime:
 UTC = timezone("UTC")
 WEEKEND = (6, 7)  # Sat, Sun
 
+SHUTDOWN_IDLE_DAYS = [
+    (1, 2),
+    (1, 3),
+    (1, 4),
+    (1, 5),
+    (1, 6),
+    (1, 7),
+]
+
+CHRISTMAS_HOLIDAYS = [
+    (1, 1),
+    (12, 24),
+    (12, 25),
+    (12, 26),
+    (12, 27),
+    (12, 28),
+    (12, 29),
+    (12, 30),
+    (12, 31),
+]
+
+SHUTDOWN_DAYS = CHRISTMAS_HOLIDAYS + [(1, 2)]
```

Comment:
Is there a point to having both `SHUTDOWN_DAYS` and `SHUTDOWN_IDLE_DAYS`? We never use the one without the other